### PR TITLE
High school idea

### PR DESCRIPTION
**Pull request by eloiseespel**

### CODE REVIEW

File: Timeline.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Timeline for high school 
+October 27th � When we want our new requirements document to be done
```
Comment (Nick-McGee) [inline Timeline.txt:2]: This is a good deadline to stick to, and I think we can achieve it.

File: highschol jupyter notebook idea.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+Using everglade we could great a Jupiter notebook (Minecraft themed) that is like homework/interactive lesson where students get to complete critical parts of Python code (we would have starter code and they would have to finish it to accomplish something). Would look really similar to the example notebook in the everglade presentation but for AI/ML. We would use python libraries for the AI and teach them those libraries. We need to confirm with Jeff that he wants the students to learn those libraries.
```
Comment (Nick-McGee) [inline highschol jupyter notebook idea.txt:1]: This is a good question for Jeff.

Comment (Ansper11) [inline highschol jupyter notebook idea.txt:1]: Thanks, this is a good question

File: Timeline.txt [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+October 28th � Start coding prototype 
+November 4th � Requirements are due 
+Milestone 1 � Coded prototype with basic features any AI or ML is mocked 
+Milestone 2 � Basics AI or ML using libraries + lesson is coded and functional 
+Final milestone � Polishing, AI/ML is finishes, all testing and bugs have been taken care of
```
Comment (yyht6300a) [inline Timeline.txt:7]: We can follow this timeline to achieve our goals.

**Pull request merged**